modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/cache/ResetLostPartitions.java

```diff
@@ -16,35 +16,51 @@
 
 package org.apache.ignite.internal.commandline.cache;
 
+import java.util.HashSet;
 import java.util.Set;
 import java.util.logging.Logger;
 import org.apache.ignite.internal.client.GridClient;
 import org.apache.ignite.internal.client.GridClientConfiguration;
 import org.apache.ignite.internal.commandline.AbstractCommand;
 import org.apache.ignite.internal.commandline.Command;
 import org.apache.ignite.internal.commandline.CommandArgIterator;
+import org.apache.ignite.internal.commandline.CommandLogger;
+import org.apache.ignite.internal.commandline.argument.CommandArgUtils;
+import org.apache.ignite.internal.commandline.cache.argument.ResetLostPartitionsCommandArg;
+import org.apache.ignite.internal.commandline.cache.distribution.CacheDistributionGroup;
+import org.apache.ignite.internal.commandline.cache.distribution.CacheDistributionNode;
+import org.apache.ignite.internal.commandline.cache.distribution.CacheDistributionPartition;
+import org.apache.ignite.internal.commandline.cache.distribution.CacheDistributionTask;
+import org.apache.ignite.internal.commandline.cache.distribution.CacheDistributionTaskArg;
+import org.apache.ignite.internal.commandline.cache.distribution.CacheDistributionTaskResult;
 import org.apache.ignite.internal.commandline.cache.reset_lost_partitions.CacheResetLostPartitionsTask;
 import org.apache.ignite.internal.commandline.cache.reset_lost_partitions.CacheResetLostPartitionsTaskArg;
 import org.apache.ignite.internal.commandline.cache.reset_lost_partitions.CacheResetLostPartitionsTaskResult;
+import org.apache.ignite.internal.processors.cache.distributed.dht.topology.GridDhtPartitionState;
 
+import static org.apache.ignite.internal.commandline.CommandLogger.optional;
+import static org.apache.ignite.internal.commandline.CommandLogger.or;
+import static org.apache.ignite.internal.commandline.TaskExecutor.BROADCAST_UUID;
 import static org.apache.ignite.internal.commandline.TaskExecutor.executeTaskByNameOnNode;
 import static org.apache.ignite.internal.commandline.cache.CacheCommands.usageCache;
 import static org.apache.ignite.internal.commandline.cache.CacheSubcommands.RESET_LOST_PARTITIONS;
+import static org.apache.ignite.internal.commandline.cache.argument.ResetLostPartitionsCommandArg.ALL_CACHES_MODE;
 
 /**
- * Command for reseting lost partition state.
+ * Command for resetting lost partition state.
  */
 public class ResetLostPartitions extends AbstractCommand<Set<String>> {
     /** {@inheritDoc} */
     @Override public void printUsage(Logger logger) {
         String CACHES = "cacheName1,...,cacheNameN";
-        String description = "Reset the state of lost partitions for the specified caches.";
+        String description = "Reset the state of lost partitions for the specified or all affected caches.";
 
-        usageCache(logger, RESET_LOST_PARTITIONS, description, null, CACHES);
+        usageCache(logger, RESET_LOST_PARTITIONS, description, null, CACHES,
```

Comment:
In my understanding, it should be as follows: 
`usageCache(logger, RESET_LOST_PARTITIONS, description, null, or(CACHES, ALL_CACHES_MODE));`

If I am not mistaken, the current implementation only allows two options:
 - reset_lost_partitions cache_name
 - reset_lost_partitions --all

Should we allow using reset_lost_partitions without any arguments (assuming that this case equals to --all option)?

Reply:
Nothing has changed if no option is specified. In that case, it's going to use the old behavior - return nothing and no additional hints or information is printed.